tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -135,3 +135,19 @@ spec:
           # value expected by Tekton Pipelines. NOTE: If params.SSL_CERT_DIR is empty, the value
           # will contain a trailing ":" - this is ok.
           value: "/tekton-custom-certs:/etc/ssl/certs:/etc/pki/tls/certs:/system/etc/security/cacerts:$(params.SSL_CERT_DIR)"
+    - name: display
```

Comment:
How about `summary`?

This is somewhat user visible so worth thinking of a good descriptive name.

Reply:
Done!